lib/dry/core/class_attributes.rb

```diff
@@ -29,7 +30,7 @@ module ClassAttributes
       #
       #   OtherClass.one # => 1
       #   OtherClass.two # => 'two'
-      def defines(*args)
+      def defines(*args, **options)
```

Comment:
I'd say this should be `defines(*args, type: Types::Any)` and we always use types w/o conditional logic.

Reply:
@solnic absolutely in favor of that change and remove conditional logic.

Thanks for the suggestion.